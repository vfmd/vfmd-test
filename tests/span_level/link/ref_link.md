A reference [link][ref1] to a url.

A reference [link][ref2] to a url.

[ref1]: url
[ref2]: url "title"
