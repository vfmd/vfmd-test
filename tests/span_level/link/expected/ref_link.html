<p>A reference <a href="url">link</a> to a url.</p>

<p>A reference <a href="url" title="title">link</a> to a url.</p>
